Solutions/GitHub/Parsers/GitHubCodeScanningData.txt

```diff
@@ -0,0 +1,31 @@
+// GitHub Enterprise code scan Data Parser
+// Last Updated Date: June 15, 2022
+//
+//This parser parses GitHub Enterprise code scan alerts and extract the infromation from their various components. It is assumed that you are using officially supported Github webhook connector (installed from Content hub)
```

Comment:
Please include steps to deploy this parser manually as well.

Reply:
done